packages/calcite-components/src/components/dialog/dialog.tsx

```diff
@@ -161,6 +161,9 @@ export class Dialog extends LitElement implements OpenCloseComponent, LoadableCo
    */
   @property({ reflect: true }) escapeDisabled = false;
 
+  /** Allows special focus-trap configuration. */
```

Comment:
Maybe `Allows for focus trapping configuration`?

Reply:
We'll probably need to document the options as well right? `"allowOutsideClick" | "initialFocus" | "returnFocusOnDeactivate"`

Reply:
Totally. @geospatialem @DitwanP Any ideas on how we should doc items from another lib? I don't think we call out using `focus-trap` or other 3rd-party lib anywhere in the doc site.

Reply:
Yeah we don't want to call out focus-trap specifically so we can say "focus trapping"

Reply:
Maybe something like:

```
Specifies custom focus trap configuration on the component, where `"allowOutsideClick`" allows outside clicks, `"initialFocus"` enables initial focus, and `"returnFocusOnDeactivate"` returns focus when not active.
```

I'm not 100% sure if that's the true intended behavior for the options, but perhaps something to that effect?